src/models/console/consoleOperation.ts

```diff
@@ -11,6 +11,7 @@ import { ConsoleHeader } from "./consoleHeader";
 export class ConsoleOperation {
     private readonly api: Api;
 
+    public readonly id: string;
```

Comment:
Do we really need this id? `public readonly id: string;`
@harunrst is moving the portal to new Data API, where there is no `id` property anymore. You folks may want to syncup about it.

Reply:
id of operation changed to the operationName which is introduced in ConsoleOperation model